### DISCUSSION TITLE
Galnet monitor not in Spanish

### DISCUSSION THREAD

**Discussion by Transcan** [EDCD/EDDI#2480] [EDDI Standalone Q&A] [answered]
I just noticed that Galnet Monitor doesn't has a Spanish option in the dropdown menu...
Is that a bug or is intended because there is no such language for Galnet?

**Comment by Tkael** [comment, marked as answer]
We currently use feeds like https://community.elitedangerous.com/en/galnet-rss.
There is no Spanish Galnet available in that format so for a long time now Spanish hasn't been an available language for Galnet.

As noted in #2415, I recently became aware of another newer URL for Galnet feeds, including from https://www.elitedangerous.com/ES-es/rss/galnet.xml. The Galnet monitor would need to be modified to accept this new format but once done we should be able to support a Spanish Galnet. 

I'll note this down as a reason to put more emphasis on that project.
Or if you / someone you know might be interested in taking it on then please let me know. :-)

**Reply by Transcan** [reply]
I see... I never noticed Galnet RSS wasn't translated, it does In-game though. Frontier being Frontier... ¬_¬

About taking on... I wish I knew how to do it, I've never used github to push changes beyond submitting bug tickets.
I wouldn't even know how to compile the code to test that everything works.
In addition, my programming knowledge is not very advanced either and is limited to simple tasks, such as modifying scripts.

But I'll take a look at the code to learn... Who knows, maybe it's easier than I think and I can help with something.

**Reply by Tkael** [reply]
Take a look at https://github.com/EDCD/EDDI/wiki/Running-from-source and https://github.com/EDCD/EDDI/blob/develop/docs/Contributing.md, both would be helpful if you do decide to give tinkering with the code a try. :-)
